projects/libaom/build.sh

```diff
@@ -25,7 +25,9 @@ rm -rf ./*
 # overflows in the transform functions.
 extra_c_flags='-DAOM_MAX_ALLOCABLE_MEMORY=1073741824 -DDO_RANGE_CHECK_CLAMP=1'
 
-cmake $SRC/aom -DCONFIG_PIC=1 -DCONFIG_SCALABILITY=0 -DCONFIG_LOWBITDEPTH=1 \
+cmake $SRC/aom -DCMAKE_BUILD_TYPE=Release -DCMAKE_C_FLAGS_RELEASE='-O3 -g' \
```

Comment:
Please remove `O3`.

We intentionally use `O1` when compiling. `O3` can reduce the number of program edges, removing important signals when fuzzing.